matrix.d.ts

```diff
@@ -960,17 +1022,202 @@ export class Matrix extends AbstractMatrix {
    * @param array - Column to add.
    */
   addColumn(index: number, array: ArrayLike<number> | AbstractMatrix): this;
+  addColumn(array: ArrayLike<number> | AbstractMatrix): this;
 
   /**
    * Adds a new row to the matrix (in place).
    * @param index - Row index. Default: `this.rows`.
    * @param array - Row to add.
    */
   addRow(index: number, array: ArrayLike<number> | AbstractMatrix): this;
+  addRow(array: ArrayLike<number> | AbstractMatrix): this;
 }
 
 export default Matrix;
 
+export class SymmetricMatrix extends Matrix {
```

Comment:
```suggestion
export class SymmetricMatrix extends AbstractMatrix {
```

Make sure the rest of the types are up-to-date with the latest changes.

Reply:
done